### PR TITLE
Adding the possibility to add meaning to the concepts_table

### PR DESCRIPTION
**Pull request by acidroyo-UMCU**
In a different project, we have identified an specific requirement for the ConcepTION data model.

Basically, we are adding the possibility to save the meaning of any CDM table -if available- in the results of the function.

### CODE REVIEW

File: R/create_dap_specific_concept.R [hunk 1]
```diff
@@ -83,6 +84,19 @@ create_dap_specific_concept <- function(codelist, data_db, name_attachment, save
     cols_temp <- na.omit(as.character(cols[j]))
     values_temp <- toupper(na.omit(as.character(values[j])))
     value <- codelist[[j, "keep_value_column_name"]]
+    if(add_meaning == TRUE){
+      columns_db_table <- dbListFields(save_db, name_edited)
```
Comment (bvreede) [inline R/create_dap_specific_concept.R:88]: what package is `dbListFields` from?

File: R/create_dap_specific_concept.R [hunk 2]
```diff
@@ -83,6 +84,19 @@ create_dap_specific_concept <- function(codelist, data_db, name_attachment, save
...
+      meaning_column_name <- columns_db_table[str_detect(columns_db_table,'meaning')]
```
Comment (bvreede) [inline R/create_dap_specific_concept.R:89]: `str_detect` is a `stringr` function, right?

**Comment by bvreede** [comment]
Thanks for the addition! Before I go into more detail in the review, please fix the CI (I didn't check but I assume it fails because there is no update in the documentation?). In addition, it would be great to add a test for the added functionality; we can build up the test suite as we build up functionality. This also provides (one or more) use case(s) straight out of the gate, so it makes it easier to review and understand the new functionality.

File: R/create_dap_specific_concept.R (hunk 1 above)
Comment (acidroyo-UMCU) [reply R/create_dap_specific_concept.R]: DBI, i just added it in the last commit.

File: R/create_dap_specific_concept.R (hunk 2 above)
Comment (acidroyo-UMCU) [reply R/create_dap_specific_concept.R]: Indeed, added in the last commit.

**Review by bvreede — APPROVED** [review]
LGTM, minus tests ;)

**Pull request merged**